docs/pages/guides/testing-rsc.mdx

```diff
@@ -5,34 +5,36 @@ description: Learn about writing unit tests for React and React Native Server Co
 platforms: ['android', 'ios', 'web']
 ---
 
-> This guide refers to the experimental feature React Server Components which is still in development.
+import { Terminal } from '~/ui/components/Snippet';
 
-React Server Components (RSC) are a new feature in React that allows you to build components that render on the server and can be hydrated on the client.
+> This guide refers to the experimental feature React Server Components which is still in development.
 
-Expo has support for testing React Server Components in your project.
+React Server Components (RSC) is a new feature in React that allows you to build components that render on the server and can be hydrated on the client. This guide provides details on how to write unit tests for RSC in your project.
 
-## Jest Testing
+## Jest testing
 
-Unlike client-based Jest testing, React Server Components run in a Node.js environment, meaning you can more closely emulate the server-side rendering environment with Jest on its own.
+React Server Components run in a Node.js environment, unlike a client-based Jest testing environment. This means you can emulate the server-side rendering environment closely with Jest.
 
 ### Setup
 
-While standard server rendering is web-only, Expo's universal React Server Components bundle custom server renderers for each platform. This means platform-extensions like `*.ios.js` and `*.native.ts` will be resolved when writing server components for an iOS app.
+While standard server rendering is web-only, Expo's universal RSC bundles custom server renderers for each platform. For example, when writing server components for an iOS app, the platform-specific extensions such as **\*.ios.js** and **\*.native.ts** will be resolved.
```

Comment:
The "For example" is overly broad. Just the iOS part is the example. The "the" sounds extraneous to me and reads better without it.

```suggestion
While standard server rendering is web-only, Expo's universal RSC bundles custom server renderers for each platform. This means platform-specific file extensions are supported. For example, when writing server components for an iOS app, platform-specific extensions such as **\*.ios.js** and **\*.native.ts** will be resolved.
```